### ISSUE TITLE
Failed to work in horovod 0.15.2

### ISSUE THREAD

**Issue by Fangyh09** [atranitell/Synchronized-BatchNorm-PyTorch-Horovod#1]
Hello, what is your horovod version?
It failed to work in horovod 0.15.2

**Comment by Fangyh09** [comment]
It worked in horovod 0.15.2 now.

**Comment by atranitell** [comment]
@Fangyh09 well?
